frontend/packages/ceph-storage-plugin/src/constants/storage.ts

```diff
@@ -0,0 +1,3 @@
+export const STORAGE_HEALTH_QUERIES = {
```

Comment:
Why not `src/components/dashboard-page/storage-dashboard/queries.ts` just like in #1732?

```ts
export const StorageHealthQueries = {
  CEPH_STATUS_QUERY: 'ceph_health_status',
};
```

This way, you can co-locate query definitions with the corresponding Dashboard tab (in this case, `storage-dashboard`).